src/util/FioAddressUtils.ts

```diff
@@ -759,25 +759,10 @@ const buyAddressRequest = async (
         }
       }
 
-      const supportedAssets: EdgeAsset[] = []
-      const { addresses, pricing } = buyAddressResponse.success.charge
-      for (const currencyKey of Object.keys(pricing)) {
-        // const currencyCode = buyAddressResponse.success.charge.pricing[currencyKey].currency
-        const asset = fioToEdgeMap[currencyKey]
-        if (asset == null) {
-          continue
-        }
-        supportedAssets.push(asset)
-        const { pluginId, tokenId } = asset
-
-        if (paymentInfo[pluginId] == null) {
-          paymentInfo[pluginId] = {}
-        }
-        paymentInfo[pluginId][tokenId ?? ''] = {
-          amount: pricing[currencyKey].amount,
-          address: addresses[currencyKey]
-        }
-      }
+      // HACK: Hardcoded assets according to FIO's web interface, since the
```

Comment:
Did the FIO API break? Did you get confirmation from FIO that it changed?

Reply:
https://edgesecure.slack.com/archives/C01GGKCV6NM/p1714593367028079

Reply:
I thought we already knew that the api was not working, which is why the task said hard-code?